estate_account/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, Command
+
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def action_sold(self):
+        res = super().action_sold()
+
+        for record in self:
+            self.env["account.move"].create({
+                "partner_id": record.buyer_id.id,
```

Comment:
Currently, nothing prevents a property for being marked as sold without a `buyer_id`. This line will cause a failure because of that. You might want to add a check to ensure a buyer is present before marking as sold.